src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -344,6 +344,9 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
   );
 };
 
+const ensureGeneViewExists = (view: string) =>
+  !!Object.values(View).some((value) => value === view);
```

Comment:
`.some` returns a boolean. No need to typecast it to boolean again.

Reply:
✅